_targets.R

```diff
@@ -4,7 +4,7 @@ library(tibble)
 library(dplyr)
 
 options(tidyverse.quiet = TRUE)
-tar_option_set(packages = c("tidyverse", "dataRetrieval", "urbnmapr", "rnaturalearth", "cowplot", "lubridate", "leaflet", "leafpop", "htmlwidgets"))
+tar_option_set(packages = c("tidyverse", "dataRetrieval", "urbnmapr", "rnaturalearth", "cowplot", "lubridate", "leaflet", "leafpop", "htmlwidgets", "dplyr"))
```

Comment:
`dplyr` comes with `tidyverse`, so you don't _have to_ include it here! Here is everything that loads when you use `tidyverse`: https://www.tidyverse.org/packages/

Reply:
Oh nice.